sdks/python/apache_beam/utils/retry.py

```diff
@@ -176,7 +176,7 @@ def wrapper(*args, **kwargs):
               sleep_interval = retry_intervals.next()
             except StopIteration:
               # Re-raise the original exception since we finished the retries.
-              raise exn, None, exn_traceback
+              raise exn, None, exn_traceback  # pylint: disable=raising-bad-type
```

Comment:
So inside an except you should do just `raise` instead of `raise exn` so this is kind of a special case. I generally prefer following as much of the standard rules as possible